source/Commands.cpp

```diff
@@ -1,5 +1,6 @@
 # include <cstring>
 # include <unistd.h>
+# include <iostream>
```

Comment:
Debug include?

Reply:
This one is apparently still useful

Reply:
Odd, It shows up as new and I can't find any std::cout or std::cerr in your code.
Do you get a compile error if you remove it? Was it perhaps in an including file originally?

Reply:
From a previous code
https://github.com/raven-os/feathers/blob/021c63fdf0f2b63cbdee1323582524da74f3e62a/source/Commands.cpp#L191

Reply:
I wonder why it compiled before. I guess we'll leave it for the moment.